src/objects/token/admin.py

```diff
@@ -104,7 +127,27 @@ def add_view(self, request, form_url="", extra_context=None):
 class PermissionInline(EditInlineAdminMixin, admin.TabularInline):
     model = Permission
     fk_name = "token_auth"
-    fields = ("object_type", "mode", "use_fields", "fields")
+    fields = (
+        "object_type",
+        "mode",
+        "use_fields",
+        "fields",
+        "get_uuid",
+        "get_version",
+    )
+
+    def get_uuid(self, obj):
+        if obj:
```

Comment:
Why do you need this condition? It can be removed. This comment is applied for all admin fields